api/search/views.py

```diff
@@ -335,16 +337,21 @@ def create_search_keys(self):
                 self.all_terms.append(keyword)
 
     # takes article ids from mongodb with its keyword
-    def get_article_ids(self, combination):
+    def get_article_ids_from_annotations(self, keyword):
         query = {}
-        query["body.value.id"] = combination
-        document = self.annotation_column.find(query)
+        #query over all keywords into search term
+        sub_keyword_strings=keyword.split(" ")
```

Comment:
this will return everything that includes "left" if one searches for "left amygdala", no?

Reply:
I removed substring operation due to our customer request.